### PR TITLE
Allow more user friendly condition names

### PR DESCRIPTION
**Pull request by wow-such-code**
Fixes issue where regex validation would crash the portlet if condition names were too complex

### CODE REVIEW

File: src/main/java/life/qbic/portal/portlet/ProjectWizardUI.java [hunk 1]
```diff
@@ -55,7 +55,7 @@
 public class ProjectWizardUI extends QBiCPortletUI {
 
   public static boolean development = false;
-  public static boolean v3RegistrationAPI = false;
+  public static boolean v3RegistrationAPI = true;
```
Comment (KochTobi) [inline src/main/java/life/qbic/portal/portlet/ProjectWizardUI.java:58]: what is this?

File: src/main/java/life/qbic/projectwizard/views/MetadataUploadView.java [hunk 2]
```diff
@@ -1187,8 +1187,9 @@ protected void createConditionWindow(ComboBox selectionBox) {
     TextField label = new TextField();
     label.setRequired(true);
     label.setStyleName(Styles.fieldTheme);
-    RegexpValidator factorLabelValidator = new RegexpValidator("([a-z]+_?[a-z]*)+([a-z]|[0-9]*)",
-        "Name must start with a lower case letter and contain only lower case letter words, which can be connected by underscores ('_'). It can end with one or more numbers.");
+    RegexpValidator factorLabelValidator = new RegexpValidator("[a-z]+[a-z0-9_]*",
+        "The name must start with a letter and may only contain letters (a-z), "
+            + "numbers and underscores ('_').");
```
Comment (KochTobi) [inline src/main/java/life/qbic/projectwizard/views/MetadataUploadView.java:1192]:
```suggestion
        "The name may only contain lower-case letters (a-z), "
            + "numbers and underscores ('_'). It must start with a lower-case letter.");
```

File: src/main/java/life/qbic/portal/portlet/ProjectWizardUI.java (hunk 1 above)
Comment (wow-such-code) [reply src/main/java/life/qbic/portal/portlet/ProjectWizardUI.java:58]: This setting tells the registration controller to use the openbis v3 api for sample registration. it was set to false accidentally in one of the previous PRs. Both modes work and have been tested, but the old API is very slow as it uses ETL processes.

**Review by KochTobi — APPROVED** [review]
LGTM

**Pull request merged**